src/ProfessionNPC.cpp

```diff
@@ -1,6 +1,7 @@
 #include "ScriptMgr.h"
 #include "Player.h"
 #include "ScriptedGossip.h"
+#include "loader.h"
```

Comment:
No need this line

Reply:
I used to think it was necessary. In fact other modules gave me an error for not using the include. But well, if it is not necessary then I remove it. Thanks for the information.